target/linux/ipq806x/patches-5.10/096-34-drivers-net-dsa-qca8k-add-support-for-multi-cpu-supp.patch

```diff
@@ -0,0 +1,117 @@
+From 2bc095a9d03fb11701ecabe9313962aa9ce42aea Mon Sep 17 00:00:00 2001
```

Comment:
This patch should be in generic, no?

Reply:
well if we test and check that qca8k works also for other target i think we should move everything to generic

Reply:
I know ath79 has hardware with supported switches. I think there was another target. Forgot which.

I'll try looking into getting it working with my archer. Probably requires ethernet driver changes though...

Reply:
If you notice we are missing some special value from the values set, just tell me and I will check with the datasheet